RecoBTag/ONNXRuntime/python/pfHiggsInteractionNet_cff.py

```diff
@@ -0,0 +1,31 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoBTag.FeatureTools.pfDeepBoostedJetTagInfos_cfi import pfDeepBoostedJetTagInfos as _pfHiggsInteractionNetTagInfos
+from RecoBTag.ONNXRuntime.pfDeepBoostedJetTags_cfi import pfDeepBoostedJetTags as _pfHiggsInteractionNetTags
```

Comment:
in this context the renaming to ```_pfHiggsInteractionNetTagInfos``` is just an obfuscation, I suggest to use ```pfDeepBoostedJetTagInfos``` and ```pfDeepBoostedJetTags``` directly

Reply:
For some reason, using them directly results in some conflict when running multiple taggers, with an error like:
```
----- Begin Fatal Exception 02-Jul-2020 22:36:29 PDT-----------------------
An exception of category 'Configuration' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing module: class=DeepBoostedJetONNXJetTagsProducer label='pfDeepBoostedJetTags'
Exception Message:
MissingParameter: Parameter 'input_names' not found.
----- End Fatal Exception -------------------------------------------------
```
I've left the renaming for now (just by putting a `_` in front).